src/vivarium_public_health/results/mortality.py

```diff
@@ -62,13 +62,8 @@ class MortalityObserver(PublicHealthObserver):
 
     def __init__(self) -> None:
         super().__init__()
-        self.required_death_columns = ["alive", "exit_time", "cause_of_death"]
-        self.required_yll_columns = [
-            "alive",
-            "cause_of_death",
-            "exit_time",
-            "years_of_life_lost",
-        ]
+        self.required_death_columns = ["exit_time"]
+        self.required_yll_columns = ["exit_time", "years_of_life_lost"]
```

Comment:
Nit: I think we should do away with these attributes. They're only used in one place, and defining them up here makes it harder to read the observer registrations and know what columns it's really using.